modules/460-log-shipper/crds/doc-ru-cluster-log-destination.yaml

```diff
@@ -71,6 +71,12 @@ spec:
                       description: Имя pipeline, который будет применён к данным перед записью в индекс.
                     endpoint:
                       description: Адрес инстанса elasticsearch.
+                    dataStreamEnabled:
+                      description: |
+                        Использовать Datastream для хранения логов (https://www.elastic.co/guide/en/elasticsearch/reference/master/data-streams.html).
+                      docTypeSupport:
```

Comment:
Hmm, it looks like there is an error with the indent.

Reply:
fixed